sale_distribute/models/sale_order_line_share_wizard_line.py

```diff
@@ -0,0 +1,18 @@
+from odoo import models, fields, api
+
+
+class SaleOrderLineShareWizardLine(models.TransientModel):
+    _name = "sale.order.line.share.wizard.line"
+    _description = "Sale Order Line Share Wizard Line"
+
+    wizard_id = fields.Many2one("sale.order.line.share.wizard", string="Wizard")
+    sale_order_line_id = fields.Many2one(
+        "sale.order.line", string="Sale Order Line", store=True
+    )
+    is_selected = fields.Boolean(string="Selected", store=True)
+    amount = fields.Float(string="Amount")
+
+    @api.onchange("is_selected", "wizard_id")
+    def _onchange_is_selected(self):
+        if not self.is_selected:
+            self.amount = 0
```

Comment:
amount is float field so it should be 0.0
```suggestion
            self.amount = 0.0
```